lab1.js

```diff
@@ -118,10 +133,31 @@ var tooHungryDay;
  meals)
 */
 
-// your code goes here
+function average (data) {
+	var total = 0;
+	for (var i = 0; i < data.length; i++) {
+		total += data[ i ];
+	}
+	var output = total / (data.length);
+	console.log(output + " is the current average");
+	return output;
+}
+
+function outputDeathDay (schedule) {
+	var storage = [];
+	for (var i = 0; i < schedule.length; i++) {
+		storage.push(schedule[i]);
+		if (average(storage) < 4 ) {
```

Comment:
this totally works...and you could make it simpler. If you have one variable that keeps a running total (and you just add to it with each iteration of the for loop), and another variable for days that you increment, you can divide running total/days.
All in all, though, I'm glad to see you wrapping your code in functions.